<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtChip8EmulatorClass</class>
 <widget class="QMainWindow" name="QtChip8EmulatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtChip8Emulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionRun"/>
   <addaction name="actionOpenDebugger"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="QtChip8Emulator.qrc">
     <normaloff>:/QtChip8Emulator/icons/file.svg</normaloff>:/QtChip8Emulator/icons/file.svg</iconset>
   </property>
   <property name="text">
    <string>Open ROM</string>
   </property>
  </action>
  <action name="actionOpenDebugger">
   <property name="icon">
    <iconset resource="QtChip8Emulator.qrc">
     <normaloff>:/QtChip8Emulator/icons/bug.svg</normaloff>:/QtChip8Emulator/icons/bug.svg</iconset>
   </property>
   <property name="text">
    <string>Debugger</string>
   </property>
   <property name="toolTip">
    <string>OpeDebugger</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="QtChip8Emulator.qrc">
     <normaloff>:/QtChip8Emulator/icons/debug-start.svg</normaloff>:/QtChip8Emulator/icons/debug-start.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtChip8Emulator.qrc"/>
 </resources>
 <connections/>
</ui>
